internal/server/server.go

```diff
@@ -379,17 +375,16 @@ func StartServer(database knowledge.GraphDB, schemaPersistor schema.Persistor,
 	r.HandleFunc("/api/query", postQueryHandler).Methods("POST")
 	r.PathPrefix("/").Handler(http.FileServer(http.Dir("./web/build/")))
 
-	bindInterface := fmt.Sprintf(":%d", viper.GetInt32("port"))
-	fmt.Printf("Listening on %s\n", bindInterface)
+	fmt.Printf("Listening on %s\n", listenInterface)
 
 	var err error
 	if viper.GetString("tls_cert") != "" {
 		fmt.Println("Server is using TLS, the connection is secure")
-		err = http.ListenAndServeTLS(bindInterface, viper.GetString("tls_cert"),
+		err = http.ListenAndServeTLS(listenInterface, viper.GetString("tls_cert"),
 			viper.GetString("tls_key"), r)
 	} else {
 		fmt.Println("[WARNING] Server is NOT using TLS, the connection is not secure")
```

Comment:
Here I would put the messge with how fixing it:
```
fmt.Printf("[WARNING] Listening on %s without TLS, use `tls_cert` option to setup a certificate\n", listenInterface)
```

Reply:
+1